src/main/io/gps.c

```diff
@@ -1028,7 +1042,7 @@ void gpsConfigureNmea(void)
                     // disable GSV MESSAGES
                     // *** THIS COMMAND FAILS TO DISABLE GSV MESSAGES WHEN SENT ****
                     // bug?? why??
-                    serialPrint(gpsPort, "$PSRF103,03,00,00,01*27\r\n");  // disable GSV (Sat info) messages
+                    serialPrint(gpsPort, GPS_PSRF_DISABLE_GSV);  // disable GSV (Sat info) messages
```

Comment:
MTK uses PMTK314 to disable GSV afaik.

Reply:
Yes, MTK uses their proprietary commands, and they only work on MTK-compatible GPS modules.